src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java

```diff
@@ -18,4 +20,18 @@ public String sayHello() {
         return helloWorldService.sayHello();
     }
 
-}
+    @DeleteMapping("{idWallet}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    void deleteSingleWallet(@PathVariable @Min(1) Long idWallet) {
+
+        try {
```

Comment:
I think, that this try-catch block should be located in service. Controller should not contain any logic or exception handling. (You can take a look at @ControllerAdvice annotation)

Reply:
this is corrected